README.md

```diff
@@ -27,23 +27,24 @@ This is the starting point for joining and contributing to the Elyra community -
 improving code, giving talks etc.
 
 
-### Dev Meetings
-Join us for our scrum to discuss development items you're working on or have questions about. Everyone is welcome and participation is optional. Note: Currently the Elyra community call is co-located with the CLAIMED community call.
+### Dev Meetings Elyra General
+Join us to discuss development items you're working on or have questions about. Everyone is welcome and participation is optional. Note: Currently the Elyra community call is co-located with the CLAIMED community call.
 
-**When**: Every Friday at [8:30AM Pacific](https://www.thetimezoneconverter.com/?t=8%3A30%20am&tz=San%20Francisco&)
+**When**: Every Friday at [8:30-9:00 AM Pacific](https://dateful.com/convert/san-francisco-california?t=0830)
 
-**Where**: [Webex](https://ibm.webex.com/meet/romeo.kienzler1)
+**Where**: [Zoom](https://zoom-lfx.platform.linuxfoundation.org/meeting/94379806729?password=f2c8ef12-518f-41ee-a64b-c169b9e9a2bb)
```

Comment:
Just tried opening that link, and got a

>**Meeting Not Found**
>
> Sorry, we couldn't find the meeting you were looking for or the meeting has ended.
>
>Please check the meeting ID and try again.
>
>If you continue to have issues, please [contact support](https://jira.linuxfoundation.org/plugins/servlet/theme/portal/4).

![image](https://github.com/user-attachments/assets/80fd2eec-2d46-4691-8271-858d677b7f27)